src/app/pagenotfound/pagenotfound.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="overflow-hidden">
+  <div fxLayout="column" fxLayoutAlign="space-around" fxLayoutGap="4rem" class="mb-5 w-100">
+    <div class="about-dockstore">
+      <div fxLayout="row" fxLayout.lt-md="column" LayoutAlign="space-between center" fxLayoutGap="1rem" class="container">
+        <div fxLayout="column" fxLayoutAlign="space-between start" fxLayoutGap="2rem">
+          <h1>Page not found</h1>
+          <h3>
```

Comment:
Unsure about this use of `h3`... would this block of text be more appropriate as a styled `p`?

Reply:
This parallels the "about" page that I stole the styling from, 
I think it looks ok, but have no real preference.